---
layout: project
title: "Computational bioinformatics"
subtitle: An introduction to the use of simulation, modelling, processing and analysis in the computational study of biology.
location: "St Paul's Girls School"
date: "23 Jan 2023"
slides: /assets/files/mis.pm-complexity-bioinformatics.pdf
permalink: /talks/2023-bioinformatics
img: /assets/img/talks/2023-bioinformatics.png
---

_Abstract._ Simulating biological systems is of great use to understanding the dynamics and relationships in nature, from the smallest scale of bacteria and neurons, all the way to the evolution of ecosystems. Most intriguing is to observe their emergent properties, where the whole is greater than the sum of the parts, such as collaboration and synchrony. In this talk I'd like to introduce you to the different uses of computing in biology, and exemplify it with something practical, a short intro to cellular automata and simulating neurons.


<br/>
<br/>

### Further Study

A [Python Jupyter notebook](/assets/files/mis.pm-bioinformatics-notebook.zip) is also available for the lecture.

### Bibliography

- Melanie Mitchell, _Complexity: A guided tour_ (2011). For an introduction to the history of complexity science and the fields is collaborates with, such as biology, genetic algorithms, evolution, cellular automata etc. Very good book to start, as it requires no prior knowledge of any of the fields.

- John Holland, _Complexity: A very short introduction_ (2014). What the title says. A little more technical than the former.

- Paulien Hogeweg, _Evolution of biological complexity from special cases to general insights_, Conference for Complex Systems 2022, [[video URL]](https://piped.kavin.rocks/watch?v=_iAsg2ILWc8). This talk can be technical at times, but it still is very inspiring, and you get an idea of how people can work in bioinformatics, besides genetics research.

- Austin Brown, _Conway's game of life and three millenia of human history_. Long Now, 2013. [[URL]](https://longnow.org/ideas/conways-game-of-life-and-three-millennia-of-human-history/). An interesting article about how one can use the philosophy behind the Game of Life simulation to simulate many more things.

- _Origins of life_ [[URL]](https://www.complexityexplorer.org/courses/103-origins-of-life) course by Complexity Explorer, the school of the Santa Fe institute. They also have a [Youtube channel](https://youtube.com/channel/UC6s-1TYa-1fBrUUIGijshCQ) with lots of other complexity science and math topics for different levels of expertise. Lots of these are for a general audience with no prior knowledge of complexity, physics, or biology.

- _Foundations of Computational and Systems Biology_ [[URL]](https://web.archive.org/web/20071222091912/http://ocw.mit.edu/OcwWeb/Biology/7-91JSpring2004/LectureNotes/index.htm). A course offered by MIT for free. It is a university course so it may have more prerequisites than the former.


### Image credits

The images in the presentation have been collected from mostly related sources, i.e. bioinformatics or computational biology research articles or lab homepages. Since not all of these are open access or creative commons, those sources are linked below.


- img 1: [Clemson University](http://compbio.clemson.edu/lab/project/12/)
- img 2,4: [Utrecht University](https://tbb.bio.uu.nl/ph/)
- img 3: [New York Times](https://www.nytimes.com/2019/02/21/science/dna-hachimoji-genetic-alphabet.html)
- img 5: [The GIST](https://the-gist.org/2014/01/and-now-for-something-completely-similar/)
- img 8: [Zohreh Zakeri](https://www.researchgate.net/publication/322581932_Optimised_Use_of_Independent_Component_Analysis_for_EEG_Signal_Processing)
- img 9: [Grinsted et al](https://www.glaciology.net/publication/2004-12-24-application-of-the-cross-wavelet-transform-and-wavelet-coherence-to-geophysical-time-series/)
- img 10: [LUT University](https://www.lut.fi/en/research-groups/computer-vision-and-pattern-recognition-laboratory-cvprl)
- img 11: [Bioinformatics stack exchange](https://bioinformatics.stackexchange.com/questions/14443/single-cell-rnaseq-cell-cluster-true-cluster-or-sub-cluster/14446)
- img 12: [Astra Zeneca](https://www.researchgate.net/publication/338435077_Chemoinformatics-Driven_Approaches_for_Kinase_Drug_Discovery)
- img 13,14: [Frohlich et al](https://pubmed.ncbi.nlm.nih.gov/30826071/) TODO - entropy genes
- img 15: [Titz et al]() TODO - network
- img 16: [Fornari et al](https://www.researchgate.net/publication/337725011_Spatially-extended_nucleation-aggregation-fragmentation_models_for_the_dynamics_of_prion-like_neurodegenerative_protein-spreading_in_the_brain_and_its_connectome) - adjacency and distro
- img 18: [Zhang et al](https://www.researchgate.net/publication/326490332_Altered_intra-_and_inter-hemispheric_functional_dysconnectivity_in_schizophrenia)
- img 19: TODO modelling


